src/methods/SANA.cpp

```diff
@@ -325,7 +325,7 @@ Alignment SANA::run() {
 	{
 	    double unweightdedScore = coreFreq[i][j]/(double)coreCount[i],
 		weightedScore = weightedCoreFreq[i][j]/totalCoreWeight[i];
-	    if(weightedScore > MIN_CORE_SCORE)printf("%s %s  %lu / %lu = %.6f weighted %.6f / %.6f = %.6f\n",
+	    if(weightedScore > MIN_CORE_SCORE)printf("%s %s  %lu / %lu = %.16f weighted %.16f / %.16f = %.16f\n",
```

Comment:
Please leave these as %0.6f, not %.16f

Reply:
Yea, this was updated by someone else idk how it got into our fork. But I reverted it back.